content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -4,14 +4,22 @@ export class ContextMenu extends React.PureComponent {
   constructor(props) {
     super(props);
     this.hideContext = this.hideContext.bind(this);
+    this.onShow = this.onShow.bind(this);
     this.onClick = this.onClick.bind(this);
   }
 
   hideContext() {
     this.props.onUpdate(false);
   }
 
+  onShow() {
+    if (this.props.onShow) {
+      this.props.onShow();
+    }
+  }
+
   componentDidMount() {
+    this.onShow();
```

Comment:
Added onShow that notifies components rendering ContextMenu, it has mounted. This is used by DSLinkMenu  to check if rendering of ContextMenu has caused any horizontal scroll.